spinnaker_components/filter/filter_common.h

```diff
@@ -0,0 +1,150 @@
+/* Common filter operator code
```

Comment:
I think it may be a hangover from when I had two filter operators.  Can definitely merge those files!